doc/FAQ.md

```diff
@@ -27,25 +27,25 @@ You may have created an infinite loop with no sleep. You'll have to restart the
 - On Browser: Stick `?noScript` at the end of the URL
 - On Steam:
   - In the menu, "Reloads" -> "Reload & Kill All Scripts".
-  - If this does not work, when launching the game, use the kill all script options.
+  - If this does not work, when launching the game, use the kill all scripts option.
 
 ---
 
 ## Steam: Where is the save game located?
 
-To maintain compatibility with the web browser version, the save game is not stored as a file on your filesystem. It lives inside the localStorage of the WebKit instance. You may export the save (and should backup it!) in the option menu.
+To maintain compatibility with the web browser version, the save game is not stored as a file on your filesystem. It lives inside the localStorage of the WebKit instance. Export the save (and back it up!) in the option menu.
 
 ---
 
 ## Steam: Game won't stop / Game is shown as "Running"
 
-Due to a limitation with the way Steam tracks the game, if you launch an external link (such as documentation), Steam may keep tracking the game as "Running" even after it is closed. You should simply have to close your web browser to fix this.
+Due to a limitation with the way Steam tracks the game, if you launch an external link (such as documentation), Steam may keep tracking the game as "Running" even after it is closed. Simply close the web browser to fix this.
 
 ---
 
```

Comment:
This was fixed by #396 , which is on the steam dev branch. So this text can be removed soon. Mostly noting that for myself so I can revisit this doc when we push 2.3.0 to production.